app/views/posts/_form.html.erb

```diff
@@ -10,8 +10,8 @@
   </div>
   <%= render 'localized_fields', f: f %>
   <div class="large-12 columns">
-    <%= f.label :documents, Post.human_attribute_name(:documents)%>
-    <%= f.file_field :documents,  multiple: true, accept: "application/pdf, text/plain" %>
```

Comment:
I had to remove the accepted filetypes since I couldn't get "text/markdown" to be accepted